cmd/influx/bucket.go

```diff
@@ -58,8 +60,8 @@ func newBucketService(f Flags) (platform.BucketService, error) {
 }
 
 func bucketCreateF(cmd *cobra.Command, args []string) error {
-	if bucketCreateFlags.orgID == "" {
-		return fmt.Errorf("must specify org-id")
+	if bucketCreateFlags.orgID != "" && bucketCreateFlags.org != "" {
```

Comment:
We should guard against the situation where neither flag is set.

Add another check for `if bucketCreateFlags.orgID == "" && bucketCreateFlags.org == ""`